docker/indexer/indexer.go

```diff
@@ -1,17 +1,17 @@
 /*
 Copyright 2022 Google LLC
 
- Licensed under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License.
- You may obtain a copy of the License at
+	 Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Is this change intentional?

Reply:
Ah no, I think the auto formatter changed this, the indentation on the copyright notices are a bit weird though, so I made them all the same now.